esy-build-package/esy-core/CommandExprParser.mll

```diff
@@ -0,0 +1,83 @@
+{ include CommandExprParserSupport }
+
+let safechars   = [^ '\\' ]
```

Comment:
Wow look at this - a *legit* lexer!

Reply:
haha, yes — `ocamllex` itself is very cool, even when used as a parser, not just lexer